src/peft/peft_model.py

```diff
@@ -515,7 +515,12 @@ def get_nb_trainable_parameters(self) -> tuple[int, int]:
 
     def print_trainable_parameters(self) -> None:
         """
-        Prints the number of trainable parameters in the model.
+        Prints the number of trainable parameters in the model. Note: print_trainable_parameters() uses
+        get_nb_trainable_parameters() which is different from num_parameters(only_trainable=True).
+        get_nb_trainable_parameters() returns (trainable parameters, all paramters) of the Peft Model which includes
+        modified backbone transformer model. For techniques like LoRA, the backbone transformer model is modified in
+        place with LoRA modules. However, for prompt tuning, the backbone transformer model is unmodified.
+        num_parameters(only_trainable=True) returns number of trainable parameters of the backbone transformer model.
```

Comment:
Just to make it totally clear, let's add here that this can result in a different number :)